docs/source/markdown/podman-build.1.md

```diff
@@ -540,6 +540,10 @@ layers are not squashed.
 Squash all of the new image's layers (including those inherited from a base
 image) into a single new layer.
 
+#### **--ssh**
```

Comment:
Should this be --stdin?

Reply:
@rhatdan no this was added in buildah but not added in `man pages` CI was failing because of this. Since command was there on `podman build` but not in man pages I added this manually.

Reply:
Well I am confused.  Heading is --ssh and then comment talks about --stdin.  No meantion of --ssh?

Reply:
@rhatdan its coming from this PR https://github.com/containers/buildah/pull/3409 because of buildah sync, since new build option was added to buildah but podman would also needs man page to be updated , following `--ssh` in unrelated to current PR.

Reply:
I see this in buildah bud man page?

```
       --ssh=default|id[=socket>|<key>[,<key>]

       SSH agent socket or keys to expose to the build.  The socket path can be left empty to use the value of default=$SSH_AUTH_SOCK

       To later use the ssh agent, use the --mount flag in a RUN instruction within a Containerfile:

```

Reply:
**--ssh**=**default**|*id[=socket>|<key>[,<key>]*

SSH agent socket or keys to expose to the build.
The socket path can be left empty to use the value of `default=$SSH_AUTH_SOCK`

To later use the ssh agent, use the --mount flag in a `RUN` instruction within a `Containerfile`:

`RUN --mount=type=secret,id=id mycmd`